GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -157,6 +158,10 @@ - (void)storeEvent:(GDTCOREvent *)event
                  userInfo:@{
                    NSLocalizedFailureReasonErrorKey : @"Storage size limit has been reached."
                  }];
+      if (self.delegate != nil) {
+        GDTCORLogDebug(@"%@", @"Delegate notified that 1 event was dropped.");
```

Comment:
Should the log include identifying info about the dropped event?

Reply:
Hmm, yeah it may be helpful to provide some identifying info...

Reply:
Added in daa581d